docs/maintaining.md

```diff
@@ -20,5 +20,12 @@ This is non-exhaustive.
   - prefer using tools that do one thing well
   - prefer using tools that are ubiquitous: `jq` or `sed` instead of `prettier` or `sd`
   - write scripts to do one thing well
+  - prefer GNU versions of `sed` and other utilities
 - Scripts should use `#!/usr/bin/env bash` as their shebang
   - This increases portability on NixOS and macOS because contributors' preferred bash may not be installed in `/bin/bash`.
+
+
+## Running CI Locally
+You can run CI tools locally. 
+Scripts expect GNU versions of tooling, so you may see unexpected results on macOS.
```

Comment:
If we're helping out the macOS users with a note here, we should probably also tell Windows users that the recommended maintenance environment on that platform is via WSL 2 and the distribution of their choice.

Reply:
Oh, good point. I'll add that note in a few!